sdk/tests/deqp/modules/shared/glsLifetimeTests.js

```diff
@@ -992,11 +992,11 @@ glsLifetimeTests.OutputAttachmentTest.prototype.iterate = function() {
     var element = elementType.gen();
 
     attacher.initAttachment(0, element);
+    this.m_outputAttacher.drawAttachment(element, refSurface);
     attacher.attach(element, container);
```

Comment:
This fix doesn't look correct. There is more than just one pair of setupContainer / drawAttachment calls in this file, and it looks like the harness assumes these are called in this order.

I suggest you add a new virtual function to glsLifetimeTests.Attacher like "detachSecondaryBindings" or "detachAuxiliaryBindings" which is empty for most of the attachment types in es3fLifetimeTests.js, except for es3fLifetimeTests.BufferTfAttacher. Then add this call between the calls to setupContainer and drawAttachment, below. Will that work?